portal/client/src/components/ListView/ListView.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+
+const ListView = ({studyList = [], handleClick}) => {
```

Comment:
lets rename this component as follows 
`StudyList`
props 
`studyList` =>`listData`
`handleClick ` => `onItemClick`